### PR TITLE
Feature/share stories

### PR DESCRIPTION
**Pull request by dubhcait**
adjusted some content according to Tara's feedback and created storie page

### CODE REVIEW

File: client/src/Component/Pages/Stories/icon-m.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        <path
+          fill='#f1d44a'
+          d='M816.2 0c27.3 0 49.4 22.1 49.4 49.4 0 9.1-2.5 17.6-6.8 24.9l11.2 28.9L841.1 92c-7.3 4.3-15.8 6.8-24.9 6.8-27.3 0-49.4-22.1-49.4-49.4S788.9 0 816.2 0z'
+        />
+      </g>
+    </svg>
+  );
+}
+
+export default Icon;
```
Comment (engshorouq) [inline client/src/Component/Pages/Stories/icon-m.js]: where you use this icon ??!!

File: client/src/Component/Pages/Stories/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import "./style.css";
+import Header from "../../CommonComponent/Header";
+import Footer from "../../CommonComponent/Footer";
+import Button from "../../CommonComponent/Button";
+
+const Stories = props => {
+  return (
+    <div>
+      <Header />
```
Comment (engshorouq) [inline client/src/Component/Pages/Stories/index.js]: pass the props in the header 
```
<Header {...props} />
```

Comment (dubhcait) [reply client/src/Component/Pages/Stories/index.js]: This file is so not up to date. I pushed from hotspoting my phone. I'll give it another go

Comment (dubhcait) [reply client/src/Component/Pages/Stories/index.js]: sorry about that up to date now

**Pull request merged**